keyboards/skyloong/gk87/bl_pro/ansi/readme.md

```diff
@@ -0,0 +1,33 @@
+# GK87 backlight Digital Indicator Keyboard
+
+![GK87 BL Pro ansi PCBAS](https://i.imgur.com/Z9wxLYh.jpg)
+
+## The PCB features:
+* QMK & VIA compatibility
+* RGB Matrix backlight
+* Mechanical switches Keyboard with LED display.
+
+
+The following is the QMK Firmware for the Destop 85% keylayout -  designed by Dongguan Jizhi Electronic Technology Co., Ltd
```

Comment:
```suggestion
The following is the QMK Firmware for the Desktop 85% keylayout -  designed by Dongguan Jizhi Electronic Technology Co., Ltd
```

Reply:
Since I could not find eny mention of "destop" on manufacturers website this looks like a typo?